### DISCUSSION TITLE
LDAP auth implementation

### DISCUSSION THREAD

**Discussion by mhdaouas** [directus/directus#8805] [Feature Requests]
Anyone has implemented successfully LDAP auth with Directus and would care to explain how he proceeded? Also how the auth hook was realized?

**Comment by rijkvanzanten** [comment]
@mhdaouas We're planning on having native support for LDAP in the near future 🙂 @aidenfoxx is doing tremendous work getting that over the finish line as we speak

**Reply by aidenfoxx** [reply]
@mhdaouas I think Rijk is aiming to have it released in rc98. That's no problem from my side, as the LDAP code is done. It doesn't have a PR up at the moment since the LDAP stuff is branched off #8660 which had some core changes, but once that's resolved I think LDAP should be easy to merge.

**Reply by mhdaouas** [reply]
Yeah I know. Thank you for this tremendous work! I thought about implementing it with ldapjs and I see that you already used the same lib. Will check the code until rc98 is released.

**Reply by aidenfoxx** [reply]
The code on github is an old PR. I'll push my recent branch either tonight or tomorrow if you want to see the new stuff. 😄
